screenshots/configs/core_screenshot_config.yaml

```diff
@@ -130,10 +130,15 @@ primary:
     message: UT dashboard doesn't render in phantomjscloud unless we set a viewport
 
   WA:
-    renderSettings:
-      maxWait: 90000
-    overseerScript: page.manualWait(); await page.waitForDelay(60000); page.done();
-    message: waiting 60 sec to load WA
+    rederSettings:
```

Comment:
renderSettings typo :) also this should actually be in `requestSettings` instead of renderSettings, sorry it's wrong in a bunch of places in this file - do you mind fixing that while you're here or in a separate PR?

Reply:
Wow, thank you! Sure.